<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputView</class>
 <widget class="QDialog" name="OutputView">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../canvas_qt4.qrc">
    <normaloff>:/images/mime.png</normaloff>:/images/mime.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>More Settings</string>
    </attribute>
    <widget class="QSpinBox" name="cutoff">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Hides colour and thickness for small data values</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>% of max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>86</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Don't show
 data if &lt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showLinkNames">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Link Names</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showLinkIds">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Link IDs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showDataValues">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Data Values</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="textsize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Hides colour and thickness for small data values</string>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Font size</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Output</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Link Thickness</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Thickness
MOE</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ThicknessMOE">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="thickness_legend">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="thickness_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>111</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="thickness_max">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxThickness">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max thickness</string>
      </property>
     </widget>
     <widget class="QLabel" name="thickness_unit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Unit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Link Colour</string>
     </property>
     <widget class="QCheckBox" name="inverseColourScale">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Inverse scale</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ColourMOE">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="colour_legend">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Colour
MOE</string>
      </property>
     </widget>
     <widget class="QLabel" name="colour_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="colour_max">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="colour_unit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Unit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../canvas_qt4.qrc"/>
 </resources>
 <connections/>
</ui>
